### PR TITLE
Add Dragonereum to Status

### PR DESCRIPTION
**Pull request by maksim-tolo**
### Summary:

Add Dragonereum to the list of Dapps

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/browser/default_dapps.cljs [hunk 1]
```diff
@@ -78,7 +78,11 @@
              :photo-path  "contacts://superrare"
              :description "Buy, sell and collect unique digital creations by artists around the world"}]}
    {:title (i18n/label :t/default-dapps-fun-games)
-    :data  [{:name        "CryptoKitties"
+    :data  [{:name        "Dragonereum"
```
Comment (rachelhamlin) [inline src/status_im/ui/screens/browser/default_dapps.cljs]: It breaks up the CK listing, can it be moved below `Crypto Takeovers`? (we also need to re-alphabetize the lists)

Comment (maksim-tolo) [reply src/status_im/ui/screens/browser/default_dapps.cljs]: Done

**Review by goranjovic — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by flexsurfer** [comment]
hey @maksim-tolo could you please resolve conflicts, thanks

**Comment by SlizkovaGanna** [comment]
Generally tested:
- App is below Crypto Takeovers
- collectible is bought and manipulated in App (all transactions valid)
- tested with different mailservers/networks/settings

Checked failed tests - could not reproduce any, the problem seemed to be temporary issues with Ropsten network

Tested on iOS 12.0 and Android 7.0

**Pull request merged**